### PR TITLE
Export subscription data to the datalake

### PR DESCRIPTION
**Pull request by alexduf**
This creates one lambda (bundled once, but used by two lambda functions).

That lambdas does a full scan of the table it receives as an argument, then streams the results to s3 as a big JSON.

### CODE REVIEW

**Comment by alexduf** [comment]
Hmm, my lambda reports success but hasn't got the right to write on the bucket I have in CODE. The file isn't written.

Still the lambda reports success. So I need to understand what's up here

File: cloudformation.yaml [hunk 1]
```diff
@@ -41,7 +41,12 @@ Parameters:
   AlarmTopic:
     Type: String
     Description: The ARN of the SNS topic to send all the cloudwatch alarms to
-
+  SubscriptionBucket:
```
Comment (NathanielBennett) [inline cloudformation.yaml]: Rename 'SubscriptionExportBucket'. Makes it clearer

File: cloudformation.yaml [hunk 2]
```diff
@@ -41,7 +41,12 @@ Parameters:
...
+    Type: String
+    Description: The name of the export subscription bucket
+  UserSubscriptionBucket:
```
Comment (NathanielBennett) [inline cloudformation.yaml]: Ditto

**Comment by NathanielBennett** [comment]
Just little things

**Comment by alexduf** [comment]
Thanks, I've fixed the name of the cloudformation parameters.
I've also:

- Tested on CODE
- created the 4 associated s3 buckets in the ophan account, checked the data was exporting there

**Pull request merged**